src/userSync.js

```diff
@@ -19,7 +19,11 @@ export function newUserSync(userSyncDependencies) {
   let numAdapterBids = {};
 
   // Use what is in config by default
-  const config = userSyncDependencies.config;
+  let usConfig = userSyncDependencies.config;
+  // Update if it's (re)set
+  config.getConfig('userSync', (conf) => {
+      usConfig = conf.userSync;
```

Comment:
This line should be `usConfig = Object.assign(usConfig, conf.userSync);` otherwise you'll lose the defaults that are not explicitly passed (ie, `enabled: true` etc).

Reply:
great point